bot/helpers/legacy_uploader.py

```
@@ -117,18 +136,88 @@ async def playlist_upload(metadata, user):
+            dest_path = f"{dest_root}/{parent_dir}".rstrip("/")
+
+    copy_cmd = f'rclone copy --config ./rclone.conf "{source_for_copy}" "{dest_path}"'
+    task = await asyncio.create_subprocess_shell(
+        copy_cmd,
+        stdout=asyncio.subprocess.PIPE,
+        stderr=asyncio.subprocess.PIPE
+    )
+    stdout, stderr = await task.communicate()
+    if task.returncode != 0:
+        try:
+            LOGGER.debug(f"Rclone copy failed: {stderr.decode().strip()}")
```

Ruff (0.12.2)

`LOGGER` may be undefined, or defined from star imports

(F405)